nexus/types/src/internal_api/params.rs

```diff
@@ -94,6 +94,11 @@ pub struct PhysicalDiskDeleteRequest {
 pub struct ZpoolPutRequest {
     /// Total size of the pool.
     pub size: ByteCount,
+
+    // Information to identify the disk to which this zpool belongs
+    pub vendor: String,
```

Comment:
nit: prefix here, like "disk_vendor"?  (it seems a little confusing to call these the "vendor", "serial", and "model" _of the zpool_)

Reply:
Updated for vendor, serial, and model.